### ISSUE TITLE
It is not possible to save an image thumb with just a width or height 

### ISSUE THREAD

**Issue by aynzad** [prismicio/slice-machine#594]
### Versions

- slice-machine-ui: 0.4.2
- node: v14.19.3

### Reproduction
using `prismicio-community/nextjs-starter-prismic-blog` as sample project

### Steps to reproduce

```
npx degit prismicio-community/nextjs-starter-prismic-blog your-project-name
cd your-project-name
npm i
npx @slicemachine/init
npm run dev

```
- Go to `http://localhost:9999/slices/Image/default`
- Click on `Edit` icon of the `Image` row
- Add a new Responsive views
- Input a name and width and left height empty
- Click on Save button

### What is expected?
to save (500xauto) and (autox500) image thumbnails without any problems, like the legacy editor.

### What is actually happening?
The save button doesn't work, Can't save the image setting that with just a fixed width or height thumbnail

<img width="956" alt="Screen Shot 2022-09-17 at 11 28 27" src="https://user-images.githubusercontent.com/3133497/190847880-7b116e28-cfed-44a4-8765-3a6aef1a0e6f.png">

<img width="521" alt="Screen Shot 2022-09-17 at 11 27 54" src="https://user-images.githubusercontent.com/3133497/190847885-01486dbc-92bf-44a1-bc8f-d8b38291f2d8.png">

**Comment by angeloashmore** [comment]
Form validation was incorrect. The new validation function checks for the following:

- A name is required.
- Only one of width or height is required. Both can be provided as well.

For the fix, see PR #601

**Comment by pavery007** [comment]
Fix released to Production (SM version 0.5.1)
